kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -63,17 +66,21 @@ public S3SinkTask() {
     // no-arg constructor required by Connect framework.
     assignment = new HashSet<>();
     topicPartitionWriters = new HashMap<>();
+    time = new SystemTime();
```

Comment:
I could issue a PR on common to synchronize them. We were using `org.apache` in the code and `io.confluent` in the Mock in the tests in this repo, until just the previous PR where I needed to pass the "clock" to the `TopicPartitionWriter` constructor.